src/routes/Landing.tsx

```diff
@@ -0,0 +1,338 @@
+                <Text as="h3" mt={5} mb={4} textStyle="landingHighlightsTitle">
+                  Anonymous
+                </Text>
+                <Text textStyle="landingHighlightsText">
+                  We don't track or store any personal information.
+                </Text>
+                <Text textStyle="landingHighlightsText" mt={4}>
+                  And the code is open source so you can always check what we
+                  are doing.
+                </Text>
+              </Box>
+              <Box maxWidth={["100%", null, "45%", "20%"]} mb={[9, 0]}>
+                <Cached height={32} />
+                <Text as="h3" mt={5} mb={4} textStyle="landingHighlightsTitle">
+                  Real time
+                </Text>
+                <Text textStyle="landingHighlightsText">
+                  Every change from your colleagues is shown instantly.
+                </Text>
+                <Text textStyle="landingHighlightsText" mt={4}>
+                  You can see how others add topics (but you can't read them
+                  until the voting step).
+                </Text>
+              </Box>
+              <Box maxWidth={["100%", null, "45%", "20%"]}>
+                <LogInCircle height={32} />
+                <Text as="h3" mt={5} mb={4} textStyle="landingHighlightsTitle">
+                  No login required
+                </Text>
+                <Text textStyle="landingHighlightsText">
+                  A retro is just a url, nothing else is needed. We don't even
+                  ask for your name or email!
+                </Text>
+                <Text textStyle="landingHighlightsText" mt={4}>
+                  And if you want to keep a retro secret, passwords are coming.
+                </Text>
+              </Box>
+            </Flex>
+          </Box>
+        </Section>
+        <Box
+          id="how-it-works"
+          py={[9, null, null, 120, 150]}
+          px={6}
+          fontSize={3}
+          lineHeight={8}
+          overflow="hidden"
+        >
+          <Box width="100ch" maxWidth="100%" margin="auto" position="relative">
+            <Screenshot display={["none", null, null, null, null, "block"]} />
+            <Box
+              width="60ch"
+              maxWidth="100%"
+              margin={["auto", null, null, null, null, 0]}
+            >
+              <Text as="h2" fontSize={[6, null, 7, 8]} fontWeight="bold">
+                How to use Retro tool
+              </Text>
+              <Text fontSize={[4, null, 5]} color="grey" my={4}>
+                The main and only screen is a 4 columns layout where you can add
+                different topics and action items (unless you are on tablet or
+                mobile, in that case the UI will adapt).
+              </Text>
+              <Text fontSize={[4, null, 5]} color="grey" mb={[8, null, 10]}>
+                A retro consists in 4 simple guided steps, you can always see
```

Comment:
As explained [here](https://english.stackexchange.com/questions/61600/consist-in-vs-consist-of), the right form would be `consists of` in this case